contribs/src/main/java/com/netflix/conductor/contribs/listener/archive/ArchivingWithTTLWorkflowStatusListener.java

```diff
@@ -29,6 +29,7 @@
 
     private static final Logger LOGGER =
             LoggerFactory.getLogger(ArchivingWithTTLWorkflowStatusListener.class);
+    private static final String ARCHIVE_OPERATION = "terminated";
```

Comment:
Did you intend to use this constant?

Reply:
ya was intended to use this to replace some duplicated strings, but forgot. Thanks for spotting that. It is fixed now